docs/quickstart.mdx

```diff
@@ -0,0 +1,209 @@
+---
+sidebar_position: 0
+sidebar_label: Quickstart
+description: "Create your first supergraph API with Hasura DDN in less than a minute."
+keywords:
+  - hasura ddn
+  - graphql api
+  - quickstart
+  - getting started
+  - guide
+  - quickstart
+hide_table_of_contents: true
+is_guide: true
+---
+
+import Step from "@site/src/components/CodeStep";
+import InstallTheCli from "@site/docs/_install-the-cli.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import SimpleVideo from "@site/src/components/SimpleVideo";
+import Admonition from "@theme/Admonition";
+
+# Quickstart with Hasura DDN
+
+<div className="flex flex-col gap-6 mb-10">
+  <div className="">
+    {/* If you break this onto another line it will lose its styling because the markdown rendering engine will add a <p> tag. Please leave it as a single line. Thanks! */}
+    <p className="text-[1.3rem]">
+      In less than <em>a minute</em> and without needing a data source connection string, you can have a supergraph API
+      running locally and deployed on Hasura DDN. 🚀{" "}
+    </p>
+  </div>
+  <div className="max-w-[750px]">
+    <SimpleVideo videoLink="https://www.youtube.com/embed/OsO6TzwFb30" />
+  </div>
+</div>
+
+## Prerequisites
+
+1. **Install the DDN CLI**
+
+   <InstallTheCli />
+
+2. **Install [Docker](https://docs.docker.com/engine/install/)** (needed for local development)
+
+   The Docker based workflow helps you iterate and develop locally without deploying any changes to Hasura DDN, making
+   the development experience faster and your feedback loops shorter. **You'll need Docker Compose `v2.27.1` or later.**
+
+3. **Validate the installation**
+
+   You can verify that the DDN CLI is installed correctly by running:
+
+   <CodeBlock language="bash">{`ddn doctor`}</CodeBlock>
+
+<Step
+id="login"
+language="bash"
+code={`ddn auth login`}
+heading={`Log in via the CLI`}
+>
+
+Authenticate your CLI session, giving you access to Hasura Cloud resources. After you log in, the CLI will acknowledge
+your login and check for any updates.
```

Comment:
Discussion:
```suggestion
Authenticate your CLI session to get access to Hasura Cloud resources. 
```
The "...achnowledge your login..." part isn't necessary to say and I checked and it doesn't check for updates any more. 

<img width="556" alt="Screenshot 2024-11-22 at 17 08 19" src="https://github.com/user-attachments/assets/563f7f3f-9748-4caa-b61b-244516c49ab3">